ompi/tools/mpirun/main.c

```diff
@@ -0,0 +1,117 @@
+/***************************************************************************
```

Comment:
shouldn't the license be at the top, not the bottom?

Reply:
I left it as it was in prior OMPI versions. IIRC, it had to do with a splash shield for debuggers